onnxruntime/test/testdata/transform/fusion/embed_layer_norm_gen.py

```diff
@@ -355,5 +354,5 @@ def GenerateModel7(model_name):
 GenerateModel3('embed_layer_norm_format3_no_cast.onnx', False)
 GenerateModel5('embed_layer_norm_format5.onnx')
 GenerateModel6('embed_layer_norm_format6.onnx')
-GenerateModel7('embed_layer_norm_format7.onnx') #distilbert
+GenerateModel7('embed_layer_norm_format7.onnx') #distilbert & shape nodes integration
```

Comment:
Could you keep format7 and add a new onnx file (like _format8.onnx) for testing merged shape?